src/modules/auth/ui/views/sign-in-view.tsx

```
@@ -0,0 +1,175 @@
+    </Button>
+    <Button 
+    disabled={pending}
+    variant="outline"
+    type="button"
+    className="w-full"
+    >
+        Github
+    </Button>
+    </div>
+    <div className="text-center text-sm">
+         Don't have an account?{" "}<Link href="/sign-up" className="underline underline-offset-4">Sign up</Link>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)